beta/src/pages/learn/index.md

```diff
@@ -410,7 +434,7 @@ export default function MyApp() {
 }
 ```
 
-Then, *pass down* the state and the event handler that's now inside `MyApp` to `MyButton`. You can pass information to `MyButton` using the JSX curly braces, just like you previously did with built-in tags like `<img>`:
+Then, *pass the state down* from `MyApp` to each `MyButton`, together with the shared click handler. You can pass information to `MyButton` using the JSX curly braces, just like you previously did with built-in tags like `<img>`:
```

Comment:
Maybe a good opportunity to say "as props" to show how state becomes props when passed down?

```suggestion
Then, *pass the state down* from `MyApp` to each `MyButton`, together with the shared click handler, as props. You can pass information to `MyButton` using the JSX curly braces, just like you previously did with built-in tags like `<img>`:
```

Reply:
We haven't introduced the words "props" yet. I wanted to wait with introducing it until we've already intuitively explained the concept. But I'm going to slightly rearrange this though.